parcellab/common/templates/_pod.tpl

```diff
@@ -175,11 +171,11 @@ spec:
       {{- end }}
   {{- if $extraContainers }}
   {{- range $extraContainers }}
-    - {{- include "common.container" (merge (deepCopy .) (dict "volumes" $podVolumes "containerEnv" $containerEnv "datadog" $datadog "commonExternalSecret" $commonExternalSecret "commonConfig" $commonConfig "commonRefName" $fullname)) | nindent 6 }}
+    {{- include "common.container" (merge (deepCopy .) (dict "volumes" $podVolumes "containerEnv" $containerEnv "datadog" $datadog "commonExternalSecret" $commonExternalSecret "commonConfig" $commonConfig "commonRefName" $fullname)) | nindent 4 }}
   {{- end }}
   {{- end }}
   {{- if or (eq $type "cronjob") (eq $type "job") }}
-  restartPolicy: {{ default "Never" .pod.restartPolicy }}
+  restartPolicy: {{ default "OnFailure" .pod.restartPolicy }}
```

Comment:
Why this change? I put it as Never because otherwise the failed pods are removed and then we lose the logs to troubleshoot those jobs